packages/circuit-ui/components/Calendar/components/CalendarWrapper/__snapshots__/CalendarWrapper.spec.js.snap

```diff
@@ -1,3 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`CalendarWrapper should render with default styles 1`] = `<div />`;
+exports[`CalendarWrapper should render with default styles 1`] = `
+<div>
+  <div />
+</div>
+`;
```

Comment:
👀  doesn't look like default styles to me

Reply:
No, I think it's fine (exactly because the component will be rebuilt from the ground up)! I even thought we could leave the useless test as-is (hence my approval), but thanks a lot for improving it!